src/NServiceBus.Core/Logging/StringBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Text;
+
+    static class StringBuilderExtensions
+    {
+        /// <summary>
+        /// Appends a new line marker to the StringBuilder followed by the text in the <paramref name="newLine"/>.
+        /// </summary>
+        public static StringBuilder NewLine(this StringBuilder stringBuilder, string newLine)
+        {
+            return stringBuilder.Append(Environment.NewLine).Append(newLine);
+        }
+
+        /// <summary>
+        /// Appends a new line marker to the StringBuilder followed by the text in the <paramref name="newLine"/>.
+        /// </summary>
+        public static StringBuilder NewLine(this StringBuilder stringBuilder, StringBuilder newLine)
+        {
+            return stringBuilder.Append(Environment.NewLine).Append(newLine);
```

Comment:
Similar to the above question
```suggestion
            return stringBuilder.AppendFormat("{0}{1}", Environment.NewLine, newLine);
```